src/test/kotlin/ractingCar/lib/ValidationTest.kt

```diff
@@ -0,0 +1,25 @@
+package ractingCar.lib
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+internal class ValidationTest {
+
+    @Test
+    fun `"3"을 입력하면 3이 나온다`() {
```

Comment:
```suggestion
    fun `정수를 입력하면 정수가 나온다`() {
```
3 대신 정수라는 의미가 더 명확해보여요.

Reply:
"String 형태의 숫자를 입력하면 정수가 나온다"로 수정하면 어떨까요?!